src/Bundle/Controller/ResourcesCollectionProvider.php

```diff
@@ -58,7 +58,8 @@ public function get(RequestConfiguration $requestConfiguration, RepositoryInterf
                 $requestConfiguration->getPaginationMaxPerPage(),
                 $paginationLimits
             ));
-            $paginator->setCurrentPage($request->query->get('page', 1));
+            $currentPage = (int) $request->query->get('page', '1');
```

Comment:
Woulndn't it be better?
```suggestion
            $currentPage = (int) $request->query->getInt('page', '1');
```

Reply:
@lchrusciel Yes, but, if it returns an integer, we don't need the (int) conversion after :)

Reply:
I pushed another suggestion to remote the unnecessary int conversion.